packages/netsuite-adapter/src/client/file_cabinet_utils.ts

```diff
@@ -0,0 +1,120 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import _ from 'lodash'
+import osPath from 'path'
+import { FILE_CABINET_PATH_SEPARATOR } from '../constants'
+
+type FileToSize = Record<string, number>
+
+type FolderSize = {
+  path: string
+  size: number
+  folders: FolderSize[]
+}
+
+type FolderSizeMap = Record<string, FolderSize>
+const sep = FILE_CABINET_PATH_SEPARATOR
+const BYTES_IN_GB = 1024 ** 3
+
+export const excludeLargeFolders = (
+  files: FileToSize,
+  maxFileCabinetSize: number
```

Comment:
I think that it would be better to be more explicit with the var name here so we could easily understand the size unit here
```suggestion
  maxFileCabinetSizeInGB: number
```